src/screens/Messages/Conversation/index.tsx

```diff
@@ -56,18 +55,6 @@ function Inner() {
   const convo = useConvo()
   const {_} = useLingui()
 
-  if (
-    convo.status === ConvoStatus.Uninitialized ||
-    convo.status === ConvoStatus.Initializing
-  ) {
-    return (
-      <CenteredView style={a.flex_1} sideBorders>
-        <Header />
-        <ListMaybePlaceholder isLoading />
-      </CenteredView>
-    )
-  }
```

Comment:
Commented on other PR already, but want to discuss this a bit. Not necessarily a blocker but want to be sure we can't do this another way.